plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2MavenPathParser.java

```diff
@@ -110,16 +118,33 @@ else if (str.endsWith("." + HashType.MD5.getExt())) {
           snapshotBuildNumber
               .append(str.substring(vSnapshotStart + version.length(), vSnapshotStart + version.length() + 8));
 
+          try {
+            final SimpleDateFormat df = new SimpleDateFormat("yyyyMMdd.HHmmss-"); // TODO: trailing "-"
```

Comment:
What is that trailing '-' for? It does not appear to match the previously existing pattern(i.e. https://github.com/sonatype/nexus-oss/blob/NEXUS-8305-maven2-md-rebuild/components/nexus-core/src/main/java/org/sonatype/nexus/proxy/maven/DefaultMetadataManager.java#L324-324)
Also, we should probably be specifying a Locale to go along with this?